2018-02-06 14:31:58,337 - main.py:22 - INFO - main - <module> - 本次测试开始执行，以下是详细日志
2018-02-06 14:32:03,526 - test_add_customer.py:15 - INFO - test_add_customer - setUp - 打开浏览器成功
2018-02-06 14:32:14,409 - test_add_customer.py:20 - INFO - test_add_customer - tearDown - 关闭浏览器成功
2018-02-06 14:32:14,464 - main.py:48 - INFO - main - <module> - 测试顺利结束^_^ 
2018-02-06 14:35:33,719 - main.py:22 - INFO - main - <module> - 本次测试开始执行，以下是详细日志
2018-02-06 14:35:37,924 - test_add_customer.py:15 - INFO - test_add_customer - setUp - 打开浏览器成功
2018-02-06 14:35:52,693 - common_logic.py:57 - INFO - common_logic - add_customer - test data is : name customer_2466,phone 18054865836, email customer_2466@qq.com
2018-02-06 14:36:09,592 - test_add_customer.py:20 - INFO - test_add_customer - tearDown - 关闭浏览器成功
2018-02-06 14:36:09,594 - main.py:48 - INFO - main - <module> - 测试顺利结束^_^ 
2018-02-06 14:38:16,965 - main.py:22 - INFO - main - <module> - 本次测试开始执行，以下是详细日志
2018-02-06 14:38:21,322 - login_logout.py:36 - INFO - login_logout - test_user_login_test - test_admin_login_test start....
2018-02-06 14:38:31,999 - login_logout.py:46 - INFO - login_logout - test_user_login_test - test data is : , 123456, 登录失败
2018-02-06 14:38:32,319 - login_logout.py:48 - INFO - login_logout - test_user_login_test - test_user_login_test end....
2018-02-06 14:38:40,891 - login_logout.py:36 - INFO - login_logout - test_user_login_test - test_admin_login_test start....
2018-02-06 14:38:47,748 - login_logout.py:46 - INFO - login_logout - test_user_login_test - test data is : invalid, 123456, 登录失败
2018-02-06 14:38:48,037 - login_logout.py:48 - INFO - login_logout - test_user_login_test - test_user_login_test end....
2018-02-06 14:38:55,470 - login_logout.py:36 - INFO - login_logout - test_user_login_test - test_admin_login_test start....
2018-02-06 14:39:02,087 - login_logout.py:46 - INFO - login_logout - test_user_login_test - test data is : admin, 123456, 退出
2018-02-06 14:39:02,390 - login_logout.py:48 - INFO - login_logout - test_user_login_test - test_user_login_test end....
2018-02-06 14:39:05,489 - main.py:48 - INFO - main - <module> - 测试顺利结束^_^ 
2018-02-06 15:06:02,459 - main.py:22 - INFO - main - <module> - 本次测试开始执行，以下是详细日志
2018-02-06 15:06:06,762 - login_logout.py:36 - INFO - login_logout - test_user_login_test - test_admin_login_test start....
2018-02-06 15:06:13,966 - login_logout.py:46 - INFO - login_logout - test_user_login_test - test data is : , 123456, 登录失败
2018-02-06 15:06:14,280 - login_logout.py:48 - INFO - login_logout - test_user_login_test - test_user_login_test end....
2018-02-06 15:06:21,722 - login_logout.py:36 - INFO - login_logout - test_user_login_test - test_admin_login_test start....
2018-02-06 15:06:28,767 - login_logout.py:46 - INFO - login_logout - test_user_login_test - test data is : invalid, 123456, 登录失败
2018-02-06 15:06:29,079 - login_logout.py:48 - INFO - login_logout - test_user_login_test - test_user_login_test end....
2018-02-06 15:06:36,529 - login_logout.py:36 - INFO - login_logout - test_user_login_test - test_admin_login_test start....
2018-02-06 15:06:43,897 - login_logout.py:46 - INFO - login_logout - test_user_login_test - test data is : admin, 123456, 退出
2018-02-06 15:06:44,210 - login_logout.py:48 - INFO - login_logout - test_user_login_test - test_user_login_test end....
2018-02-06 15:06:47,288 - main.py:48 - INFO - main - <module> - 测试顺利结束^_^ 
2018-02-06 15:15:36,968 - main.py:22 - INFO - main - <module> - 本次测试开始执行，以下是详细日志
2018-02-06 15:15:43,297 - create_new_clients.py:14 - INFO - create_new_clients - setUp - 打开浏览器成功
2018-02-06 15:15:43,297 - create_new_clients.py:29 - INFO - create_new_clients - test_create_new_clients - test_create_new_clients start....
2018-02-06 15:15:57,804 - common_logic.py:57 - INFO - common_logic - add_customer - test data is : name customer_7432,phone 13898436406, email customer_7432@qq.com
2018-02-06 15:16:15,112 - create_new_clients.py:19 - INFO - create_new_clients - tearDown - 关闭浏览器成功
2018-02-06 15:16:15,113 - main.py:48 - INFO - main - <module> - 测试顺利结束^_^ 
